pkgs/applications/misc/barrier/default.nix

```diff
@@ -13,9 +13,17 @@ stdenv.mkDerivation rec {
   buildInputs = [ cmake curl xorg.libX11 xorg.libXext xorg.libXtst avahiWithLibdnssdCompat ];
   propagatedBuildInputs = with qt5; [ qtbase ];
 
+  patches = [
+    # Fix compilation on Qt 5.11
```

Comment:
You should add a note that this can be removed on the next version, since the commit is in master already, see https://github.com/debauchee/barrier/issues/49

Reply:
Done! Thanks for review~